src/foam/nanos/auth/User.js

```diff
@@ -528,6 +528,7 @@ foam.CLASS({
       section: 'operationsInformation',
       order: 80,
       gridColumns: 6,
+      readPermissionRequired: true,
```

Comment:
can we overwrite it on nanopay side instead?

Reply:
Will do